dhis-2/dhis-web/dhis-web-commons/src/main/java/org/hisp/dhis/about/action/RedirectAction.java

```diff
@@ -77,13 +77,21 @@ public String execute()
                     if ( app.getShortName().equals( startModule.substring( "app:".length() ) ) )
                     {
                         redirectUrl = app.getLaunchUrl();
+                        if ( redirectUrl.endsWith( "dhis-web-dataentry/" ) )
```

Comment:
Could we move `dhis-web-dataentry` to a final variable and re-use?

Reply:
now there is only one occurence.